views/templates/components/progress-bar.html.twig

```diff
@@ -0,0 +1,10 @@
+<div data-component="progress-bar" class="progress log-progress__bar">
+    <div class="progress-bar"
+         role="progressbar"
+         data-title-template="{{ "Progress is {progress_percentage}%"|trans() }}"
```

Comment:
I see an empty object sent in all other calls to `trans`, isn't this line triggering any issue?

Reply:
I don't want the content of the {} to be replaced by trans but by the JS